docs/patterns/modal.md

```diff
@@ -13,6 +13,12 @@ The modal component can be used to overlay an area of the screen which can conta
 View example of the modal pattern
 </a>
 
+<div class="p-notification--caution">
+  <p class="p-notification__response">
+    <span class="p-notification__status">Warning:</span>This pattern requires JS to be functional.
+  </p>
+</div>
+
```

Comment:
This pattern is missing information on how to close the model. We should explain what the `showModal` function is doing.

Reply:
Added 👍